### PR TITLE
[MAINTENANCE] Use YAMLHandler in more places, relates to #7507

### PR DESCRIPTION
**Pull request by kenwade4**
Changes proposed in this pull request:
- Use YAMLHandler in more places; relates to #7507

### CODE REVIEW

File: tests/datasource/fluent/test_viral_snippets.py [hunk 1]
```diff
@@ -218,8 +218,11 @@ def test_file_context_add_and_save_fluent_datasource(
 
     assert datasource_name == ds.name
     assert datasource_name in final_yaml
-    # ensure comments preserved
-    assert "# Welcome to Great Expectations!" in final_yaml
+    assert "config_version" in final_yaml
+    assert "fluent_datasources:" in final_yaml
+
+    # # ensure comments preserved
+    # assert "# Welcome to Great Expectations!" in final_yaml
```
Comment (Kilo59) [inline tests/datasource/fluent/test_viral_snippets.py:143]: Any idea why comments aren't being preserved?

File: great_expectations/datasource/fluent/fluent_base_model.py [hunk 2]
```diff
@@ -64,7 +61,7 @@ class Config:
 
     @classmethod
     def parse_yaml(cls: Type[_Self], f: Union[pathlib.Path, str]) -> _Self:
-        loaded = yaml.load(f)
+        loaded = yaml.load(f)  # type: ignore[arg-type]
```
Comment (Kilo59) [inline great_expectations/datasource/fluent/fluent_base_model.py:64]: I'm somewhat inclined to remove the return type for `YAMLHandler.load`, it's not very helpful and causes a lot of issues.

**Comment by molliemarie** [comment]
Hello @kenwade4. With the launch of Great Expectations Core (GX 1.0), we are closing old PRs posted regarding previous versions. Moving forward, we will focus our resources on supporting and improving GX Core (version 1.0 and beyond). If you find that an issue or PR you previously reported still exists in GX Core, we encourage you to resubmit it against the new version. With more resources dedicated to community support, we aim to tackle new issues swiftly. For specific details on what is GX-supported vs community-supported, you can reference our [integration and support policy](https://docs.greatexpectations.io/docs/application_integration_support/).

To get started on your transition to GX Core, check out the [GX Core quickstart](https://docs.greatexpectations.io/docs/core/introduction/try_gx/#test-features-and-functionality) (click “Full example code” tab to see a code example).

Thank you for being part of the GX community and thank you for submitting this issue. We're excited about this new chapter and look forward to your feedback on GX Core. 🤗

**Comment by molliemarie** [comment]
Hello @kenwade4. With the launch of Great Expectations Core (GX 1.0), we are closing old PRs posted regarding previous versions. Moving forward, we will focus our resources on supporting and improving GX Core (version 1.0 and beyond). If you find that an issue or PR you previously reported still exists in GX Core, we encourage you to resubmit it against the new version. With more resources dedicated to community support, we aim to tackle new issues swiftly. For specific details on what is GX-supported vs community-supported, you can reference our [integration and support policy](https://docs.greatexpectations.io/docs/application_integration_support/).

To get started on your transition to GX Core, check out the [GX Core quickstart](https://docs.greatexpectations.io/docs/core/introduction/try_gx/#test-features-and-functionality) (click “Full example code” tab to see a code example).

Thank you for being part of the GX community and thank you for submitting this issue. We're excited about this new chapter and look forward to your feedback on GX Core. 🤗

**Pull request closed without merging**